CHANGELOG.md

```diff
@@ -153,6 +153,7 @@
 - PR #2928 CSV Reader: Fix using `byte_range` for large datasets
 - PR #2983 Fix sm_70+ race condition in gpu_unsnap
 - PR #2964 ORC Writer: Segfault when writing mixed numeric and string columns
+- PR #3007 Java tests: disable a test that expected an error from Rmm but now it doesn't get it
```

Comment:
```suggestion
- PR #3007 Java: Disable unit test that frees zero pointer until RMM issue is fixed
```

Reply:
It's not a zero pointer, but instead an invalid pointer (`reinterpret_cast<void*>(100)`), but I can adjust the comment.

Reply:
The new behavior of RMM is more correct. rmmFree() can't throw because it might be used in destructors, which can't throw: https://isocpp.org/wiki/faq/exceptions#dtors-shouldnt-throw

Reply:
Oh sorry I missed that this was testing a free of an invalid pointer not a zero pointer.  In that case I think this test should simply be removed.

Reply:
@jlowe @harrism test removed. Thanks!